src/InventoryManager.java

```diff
@@ -0,0 +1,263 @@
+                System.out.println("   [5] Update item price");
+                System.out.println("   [6] Quit");
+                System.out.print("Enter option: ");
+
+                // read user option from command line
+                option = br.readLine();
+                if((option == null) || (option.length() <= 0)) {
+                  System.out.println("Invalid option");
+                    option = "0";
+                    continue;
+               }
+
+                // this runs the create item functionality
+                if(option.equals("1")) {
+                   System.out.println("Enter comma delimiited name,price,quantity,category");
+                    String line = br.readLine();
+                    String[] parts = line.split(",");
+                    if((parts == null) || (parts.length != 4)) {
+                        System.out.println("Invalid inventory");
+                        continue;
+                    }
+                    String name = parts[0];
+                    String price = parts[1];
+                    String quantity = parts[2];
+                    String category = parts[3];
+
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid quantity");
+                        continue;
+                    }
+                    if(!isDouble(price)) {
+                        System.out.println("Invalid price");
+                        continue;
+                    }
+
+                    Inventory inventory = createItem(name, Double.parseDouble(price), Integer.parseInt(quantity), category);
+                    if(itemMap.containsKey(name)) {
+                        System.out.println(name + " Already created.");
+                        continue;
+
+                    }
+                    else {
+                        itemMap.put(name, inventory);
+
+                    }
+
+                }
+
+                // sell an item thus reducing the value of quatity
+                else if(option.equals("2")) {
+                    System.out.println("Enter comma delimiited name,quantity");
+                    String line = br.readLine();
+                    String[] parts = line.split(",");
+                    if((parts == null) || (parts.length != 2)) {
+                        System.out.println("Invalid inventory");
+                        continue;
+                    }
+                    String name = parts[0];
+                    String quantity = parts[1];
+
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid quantity");
+                        continue;
+                    }
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println(name + " Does not exist.");
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    int i = inventory.getQuantity();
```

Comment:
Watch out for naming variables with a single letter `currentQuantity` would be much more descriptive.